### ISSUE TITLE
Cannot scroll down Start Container Dialog

### ISSUE THREAD

**Issue by linquize** [crosbymichael/dockerui#174]
I use high DPI monitor so I press `Ctrl`+`+` to zoom-in the page.
Start Container Dialog is so tall that I need to scrolll down to press `Create` button.
However, I cannot scroll down since the Dialog is modal, the page scroll bar is disabled when the Dialog is shown.

**Comment by kevana** [comment]
What browser do you use? On Chrome the page scroll locks but as long as my cursor is over the modal I can scroll it with a trackpad or mouse-ball. Arrow keys don't work.

Stepping back, I've been considering switching this to its own page for a while. A modal made sense when there were only a few options but now there's so many options that a modal is really cramped.

**Comment by linquize** [comment]
I use Firefox. I cannot scroll with trackpad.

**Comment by deviantony** [comment]
I agree with @kevana, we should replace the modal with its own view. There's currently too much controls in that modal, it's messy.
